in-work/markov_chains.md

```diff
@@ -1125,54 +1130,48 @@ plt.show()
 ### Example 2
 
 
-However, in the case of our periodic chain, we find distributions is oscillating
+However, in the case of our periodic chain, we find the distribution is oscillating
 
 ```{code-cell} ipython3
 import random
 
 P = np.array([[0, 1],
               [1, 0]])
-n = 25
+n = 50
 n_state = P.shape[0]
 mc = qe.MarkovChain(P)
 ψ_star = mc.stationary_distributions[0]
 fig, axes = plt.subplots(nrows=1, ncols=n_state)
 x0s = np.ones((n, n_state))
 for i in range(n):
-    # Define the initial state
     nums = np.random.randint(1, 10_000_000, size=n_state)
     x0s[i,:] = np.array(nums/sum(nums))
-# Define the number of iterations
-n = 50
 
 for x0 in x0s:
-    # Define the resulting state
     x = x0
-    # Define the sum of all resulting states
-    x_sum = np.zeros(n_state)
     X = np.zeros((n,n_state))
     
-    for i in range(1, n):
+    for t in range(0, n):
         x = x @ P
-        X[i] = x
+        X[t] = x
     for i in range(n_state):
         axes[i].plot(range(20, n), X[20:,i], alpha=0.3)
     
 for i in range(n_state):
     axes[i].axhline(ψ_star[i], linestyle='dashed', lw=2, color = 'black', label = fr'$\psi (X={i})^*$')
+    axes[i].set_xlabel('t')
+    axes[i].set_ylabel(fr'$\psi(X={i})$')
     axes[i].legend()
 
 plt.show()
 ```
 
-This shows an important fact that asymptopasymptotic stationarity is about the distribution, but ergodicity is about the sample path.
+This shows an important fact that asymptotic stationarity is about the distribution, but ergodicity is about the sample path.
```

Comment:
"This example shows an important" -> "This example helps to emphasize the"
"but" -> "while"